Tests/CommandsTests/PackageCommandTests.swift

```diff
@@ -1647,7 +1655,8 @@ final class PackageCommandTests: CommandsTestCase {
             )
 
             // Invoke it, and check the results.
-            let (stdout, stderr) = try SwiftPM.Build.execute(packagePath: packageDir)
+            let args = staticStdlib ? ["--static-swift-stdlib"] : []
+            let (stdout, stderr) = try SwiftPM.Build.execute(args, packagePath: packageDir)
```

Comment:
Should this be testing produces now since the paths have diverged to make sure that they are placed correctly?

Reply:
Added the check in a lower-level test case https://github.com/apple/swift-package-manager/pull/7695/commits/ae28ac2dd706bc42bf626881693cffcbaded6f9b

Reply:
Thanks!